concordia/views.py

```diff
@@ -221,26 +221,23 @@ def post(self, *args, **kwargs):
             cmd = 'cp -r {0}/* {1}'.format('/concordia_images', collection_path)
             if os.WEXITSTATUS(os.system(cmd)) == 0:
                 os.system('rm -rf {0}'.format('/concordia_images/*'))
-            c = Collection.objects.create(title=name, slug=name.replace(" ", "-"), description=name)
-            count = 0
-            for root, dirs, files in os.walk(collection_path):
-                for filename in files:
-                    filename = os.path.join(root, filename)
-                    if True:
-                        count += 1
-                        title = '{0} asset {1}'.format(name, count)
-                        media_url = os.path.join(root, filename).replace(settings.MEDIA_ROOT, '')
+                c = Collection.objects.create(title=name, slug=name.replace(" ", "-"), description=name)
```

Comment:
This block of code still needs the various improvements in #5.

Reply:
I distrust this kind of deeply nested unexplained code since it's not obvious whether it's correct or what the error states are. For example, the `os.system` call doesn't have an error handling path and there's no indication that this is correct.

Using `subprocess.check_call` or `subprocess.check_output` would reliably raise exceptions but it's not immediately obvious what the correct behaviour would be. In this case it's also unnecessary since it would be faster and safer to use `shutil.rmtree` which also has error handling.